django/db/migrations/questioner.py

```diff
@@ -186,35 +186,35 @@ def ask_not_null_alteration(self, field_name, model_name):
 
     def ask_rename(self, model_name, old_name, new_name, field_instance):
         """Was this field really renamed?"""
-        msg = "Did you rename %s.%s to %s.%s (a %s)? [y/N]"
+        msg = "Was %s.%s renamed to %s.%s (a %s) [y/N]?"
```

Comment:
Question mark has been moved here but not in `ask_rename_model` or `ask_merge` - I think it should go back to before the `[y/N]` for consistency.